src/features/services/components/ledger-settings.tsx

```diff
@@ -38,9 +38,24 @@ function TokenRow({ name, symbol, address }: Token) {
   );
 }
 
+function toToken(token: TokenInfo): Token {
+  return {
+    name: token.info.summary.name,
+    address: token.info.address.toString(),
+    symbol: token.info.summary.symbol,
+  };
+}
+
 export function LedgerSettings() {
   const account = useAccountsStore((s) => s.byId.get(s.activeId));
-  const { data, isError, isLoading } = useTokenInfo();
+  const tokenList = useTokenList();
+  const tokenInfo = useTokenInfo(tokenList);
```

Comment:
Dependent queries. First we have to fetch the list of all tokens, then the info for each one.